### DISCUSSION TITLE
Copy ~/.config/nvim and ~/.local/share/nvim to another computer with the same OS does not work

### DISCUSSION THREAD

**Discussion by real-hong** [wbthomason/packer.nvim#1024] [Q&A]
I want to use nvim on another computer without network (same OS) as well, so I copied ~/.config/nvim and ~/.local/share/nvim from my own computer to that machine, but it doesn't work. My config file is https://github.com/the-die/dotfiles/tree/main/nvim. How can I fix it please?

**Comment by EdenEast** [comment]
What does not work? How can I reproduce your issue? I linked my  `~/.config/nvim` to your config and called `PackerSync` and it works without issue. The only thing I had to do was clone packer using the clone command in the readme.

**Reply by real-hong** [reply]
I have two computers 'a' and 'b', 'a' is able to connect to the network and 'b' is not able to connect to the network. I installed the plugin using pakcer.nvim on computer 'a' and then copied ~/.config/nvim and ~/.local/share/nvim from computer 'a' to computer 'b'. Then I run 'PackerSync' on computer 'b' and open lspsaga to get the following error.
![Screenshot from 2022-08-22 20-26-41](https://user-images.githubusercontent.com/43773170/185920880-3b6e16e9-e6af-4c50-8c6a-7d3e113202db.png)

**Reply by EdenEast** [reply]
Ah looking at that error it looks like `~/.local/share/nvim` is not in the `runtimepath`. Try adding this line to your config:

```lua
vim.opt.runtimepath:append(vim.fn.fnamemodify("~/.local/share/nvim", ":p"))
```

I would also check what the difference is between your two systems.

```lua
local file = io.open('runtimepath.log', "w")
file:write(table.concat(vim.opt.runtimepath:get(), "\n"))
file:close()
```

This writes your runtime paths into a log file that you can then check the difference between the two.

**Reply by real-hong** [reply]
Added that line to my configuration and still got the same error. The log output for both computers is as follows, the left side is the log for computer 'a' and the right side is the log for computer 'b' after adding that line of code.
![Screenshot from 2022-08-22 22-14-04](https://user-images.githubusercontent.com/43773170/185943488-42104369-23bb-4cc4-9090-b51da3783220.png)
When I type 'packadd lspsags.nvim' it works fine, but the next time I reopen nvim I still get an error.

**Reply by EdenEast** [reply]
Oh it looks like `~/.local/share/nvim/site` is in your path already so we dont have to do that additional append to the runtimepath. Can you just try and run `PackerCompile`. Since you said that the other computer is not connected to the internet the `sync` might be failing to update and not get to the `compile` step.

**Reply by real-hong** [reply]
I have run `PackerComplie`, but it doesn't work.

**Reply by EdenEast** [reply]
Have you deleted the compiled file `rm ~/.config/nvim/plugin/packer_compiled.lua` and then re-execute `:PackerCompile`?

**Reply by real-hong** [reply]
Yes, but it doesn't work.
